### PR TITLE
Update readme to reflect repo

### PR DESCRIPTION
**Pull request by rudyflores**
**What It Does**

Update readme to properly reflect how to use and install the project.

**How to Test**

**Review Checklist**
I certify that I have:
- [ ] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -3,7 +3,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```
Comment (t1m0thyj) [inline .editorconfig:6]: Is it intentional to change the indent from 2 to 4? All our source files are still using 4 spaces, and I think this is consistent with other projects like ZE.

File: README.md [hunk 2]
```diff
@@ -1,239 +1,99 @@
-# Secure Credential Store Plug-in for Zowe CLI
+# Kubernetes Secrets for Zowe CLI & Zowe Explorer
 
-<img src="imgs/Zowe_ConformanceBadge_cli.png" width=25% alt="Zowe Conformance Badge"/>
+The Kubernetes Secrets plugin for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credetials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.
```
Comment (t1m0thyj) [inline README.md:3]: `credetials` -> `credentials`

File: README.md [hunk 3]
```diff
@@ -1,239 +1,99 @@
...
 
-Any failures potentially indicate an issue with the set-up of the Rest API or configuration parameters that were passed in the `custom_properties.yaml` file.
+Follow these steps:
 
-## Uninstalling
+1. Issue the following command:
 
-**Follow these steps:**
+   ```bash
+   zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
```
Comment (t1m0thyj) [inline README.md:86]: Should the plugin name be updated to the K8s plugin?

File: .editorconfig (hunk 1 above)
Comment (rudyflores) [reply .editorconfig:6]: I saw some warnings on the readme when performing 4 indent vs 2 indent for some reason, I'm not sure if there is some issue with how the linter is viewing this vs how many tabs are made which led me to change this...

File: README.md (hunk 2 above)
Comment (anaxceron) [inline README.md:3]: please hyphenate all uses of the word **plug-in** so its consistent w/ Zowe Docs

File: README.md [hunk 4]
```diff
@@ -1,239 +1,99 @@
...
--   [Securing Zowe CLI profiles](#securing-zowe-cli-profiles)
--   [Running tests](#running-tests)
--   [Uninstalling](#uninstalling)
--   [Contributing](#contributing)
+In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.
 
+## Requirements
 
-## How the plug-in works
+Before you install and use the plug-in make sure you have the following installed:
```
Comment (anaxceron) [inline README.md:15]: please define what needs to be installed -- is it "applications"? is it "items"? Rewrite to say: Make sure you have the following applications/items/whatever installed:"

File: README.md [hunk 5]
```diff
@@ -1,239 +1,99 @@
...
-
-- Follow all installation steps above for Linux
-
-- Install the following packages:
-  - Python 3 (> 3.6.0)
-  - `make`
-  - `gcc-c++`
-  - `libsecret-devel`
+- Zowe CLI on your computer. For more information, see [installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
+- Install [Node.js](https://nodejs.org/en/download) v16.0 or later.
```
Comment (anaxceron) [inline README.md:18]: - I don't think it's necessary to include "on your computer"
- Remove the verb "install" b/c this is a list of items (according to the sentence in Line 15), not actions.

File: README.md [hunk 6]
```diff
@@ -1,239 +1,99 @@
...
-- System
-
-**Note:** For detailed information about conventions and best practices for running tests against Zowe CLI plug-ins, see see [Zowe CLI Plug-in Testing Guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/PluginTESTINGGuidelines.md).
-
-Before running the system and integration tests, you must have a server connection to run against. For more information, see [Software requirements](#software-requirements).
-
-To define access credentials to the server, copy the file named `.../__tests__/__resources__/properties/example_properties.yaml` and create a file named `.../__tests__/__resources__/properties/custom_properties.yaml`.
-
-**Note:** Information about how to customize the `custom_properties.yaml` file is provided in the yaml file itself.
+When an unsuccessful message displays, you can troubleshoot the installation by addressing the issues that the message describes. You can also review the information that is contained in the log file that is located in the Zowe CLI home directory.
```
Comment (anaxceron) [inline README.md:77]: - is it an "unsuccessful" message or is it a "failure" message?
- I would reword the sentence: " ... by addressing the issues outlined/described in the message."
- Also would tweak the next sentence: "... contained in the log file located in the Zowe CLI ..."

File: packages/cli/CHANGELOG.md [hunk 7]
```diff
@@ -5,3 +5,4 @@ All notable changes to the Kubernetes Secrets Credential Manager Plug-in for Zow
 ## Recent Changes
 
 - Added system test for v2 profiles
+- Added readme with instructions on how to use and install the repo
```
Comment (anaxceron) [inline packages/cli/CHANGELOG.md:8]: Please add a link to the related issue or PR so that we can included that in the release notes

**Review by anaxceron — APPROVED** [review]
Thanks for working on this, Rudy! Left some comments, most of them regarding style.

File: README.md (hunk 6 above)
Comment (rudyflores) [reply README.md:77]: it is a failure message that is displayed when something went wrong when installing the plug-in :)

File: .editorconfig (hunk 1 above)
Comment (rudyflores) [reply .editorconfig]: I switched it back to 4, perhaps we can see later why that warning for indents shows with 4 on md files

**Comment by rudyflores** [comment]
Thanks for the very helpful feedback @anaxceron @t1m0thyj !

File: README.md [hunk 8]
```diff
@@ -1,239 +1,99 @@
...
-### zLinux
-
-- Follow all installation steps above for Linux
-
-- Install the following packages:
-  - Python 3 (> 3.6.0)
-  - `make`
-  - `gcc-c++`
-  - `libsecret-devel`
+- Zowe CLI For more information, see [installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
```
Comment (anaxceron) [inline README.md:17]: Needs a period between "Zowe CLI" and "For more information"

**Review by anaxceron — APPROVED** [review]
Looks great, Rudy -- just one minor typo that I pointed out in a comment. Thank you!

**Review by JillieBeanSim — COMMENTED** [review]
Thanks @rudyflores for putting the readme together.  Since it is top level readme should it include installation/build instructions for both the cli and vsc ext?  Will you be adding a readme for the separate packages?  I think they are used in the marketplaces (vsc & npm) for package description and how to's

**Comment by rudyflores** [comment]
@JillieBeanSim updates were made now, I added a top level readme with common instructions between both repos, and inside each repo (vscode and cli) I have added a readme that addresses installation and other cases for each one properly.

File: README.md [hunk 9]
```diff
@@ -1,239 +1,23 @@
...
-
--   [How the plug-in works](#how-the-plug-in-works)
--   [Software requirements](#software-requirements)
--   [Installing](#installing)
--   [Building from source](#building-from-source)
--   [Securing Zowe CLI profiles](#securing-zowe-cli-profiles)
--   [Running tests](#running-tests)
--   [Uninstalling](#uninstalling)
--   [Contributing](#contributing)
+The Kubernetes Secrets plug-in for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credentials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.
```
Comment (traeok) [inline README.md:3]:
```suggestion
The Kubernetes Secrets plug-in for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credentials as Kubernetes secrets in your Kubernetes cluster of choice. It changes the default credential manager when installed in Zowe CLI or Zowe Explorer.
```

File: README.md [hunk 10]
```diff
@@ -1,239 +1,23 @@
...
 
+This plug-in can also be used as a sample to build your own custom credential manager to use based on your own use case.
```
Comment (traeok) [inline README.md:5]:
```suggestion
This plug-in also serves as a custom credential manager sample for your own use cases.
```

File: README.md [hunk 11]
```diff
@@ -1,239 +1,23 @@
...
-
--   Run the commands below to unlock the Gnome keyring. The keyring must be unlocked again for each new user session.
+The plug-in uses the Kubernetes API in order to communicate with your Kubernetes cluster and store secrets on a per user basis.
 
-    The second command will prompt for your password. Press Ctrl+D twice when you have finished typing it.
-    ```bash
-    export $(dbus-launch)
-    gnome-keyring-daemon -r --unlock --components=secrets
-    ```
+In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.
```
Comment (traeok) [inline README.md:11]:
```suggestion
In addition, when storing Kubernetes secrets, users can specify a namespace through the environment variables `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`. Otherwise, the secret will be stored in the assigned namespace for the current context.
```

File: README.md [hunk 12]
```diff
@@ -1,239 +1,23 @@
...
 
--   To automatically unlock the Gnome keyring at log on:
+## Requirements
 
-    **Note:** The following steps have been tested on CentOS, SUSE, and Ubuntu. Results may vary on other Linux distributions.
+Before you install and use the plug-in make sure you have the following tools installed:
```
Comment (traeok) [inline README.md:15]: missing comma between "plug-in" and "make sure"

**Review by traeok — APPROVED** [review]
LGTM, thanks @rudyflores - left a couple suggestions 👍

**Review by traeok — APPROVED** [review]
LGTM, thanks @rudyflores!

File: packages/cli/README.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
+# Kubernetes Secrets for Zowe CLI
+
+## Installing
+
+Use one of the following methods to install the plug-in:
+
+- Install the plug-in to Zowe CLI from an online registry or a local package.
+
+  Use this method when you want to install the plug-in quickly and start using it. With this method, you secure your user IDs and passwords manually.
```
Comment (anaxceron) [inline packages/cli/README.md:9]: When you say, "Use this method," which method are you referring to, the online registry or the local package? Or do you mean to say, "Use these methods when ..."
Should we split the online registry and local package into two methods? It might be a little confusing to have two different options w/in one method.

File: packages/vscode/README.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
+# Kubernetes Secrets for Zowe Explorer
+
+## Installing
+
+Use one of the following methods to install the plug-in:
+
+- Install the plug-in to VS Code from the marketplace
+
+  1. On the left most side of the VS Code window in your workspace, click on the extensions icon
```
Comment (anaxceron) [inline packages/vscode/README.md:9]: - Step 1 and Step 3 should end with periods.
- Any time you refer to a specific item in the UI, I would capitalize it and bold it: Click on the **Extensions** icon; Click on the Search bar; Click on the Install button. It's just a good way to let the reader know that you're referring to a specific thing on the UI.

File: packages/vscode/README.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+  Use the build from source method when you want to install the plug-in to VS Code using the most current binaries and modify the behavior of the plug-in. For example, you want to create a new command and use the plug-in with the command that you created.
+
+  For more information, see [Building from source](#building-from-source).
+
+## Building from source
+
+To build from source, follow these steps:
+
+1. Clone this repository
```
Comment (anaxceron) [inline packages/vscode/README.md:25]: Any steps that are a complete sentence, like this one, should end w/ a  period.

File: packages/vscode/README.md [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+1. Install, build, and package the project with the following command:
+
+   ```bash
+     npm install && npm build && npm package
+   ```
+
+1. View the `dist/` folder in the project, this should contain a file named `Zowe.secrets-for-kubernetes.vsix`
```
Comment (anaxceron) [inline packages/vscode/README.md:32]: I would tweak this sentence a little. Either:

- View the `dist/` folder in the project. This should contain ...
- View the `dist/` folder in the project, which should contain ...

File: packages/vscode/README.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+
+## Uninstalling
+
+To uninstall the plug-in, perform the following steps:
+
+1. On the left most side of the VS Code window in your workspace, click on the extensions icon
+
+1. On top of the extensions view, click on the search bar and search for `Kubernetes Secrets for Zowe Explorer`
+
+1. click uninstall
```
Comment (anaxceron) [inline packages/vscode/README.md:43]: All the Uninstalling steps should end w/ periods.

File: packages/cli/README.md (hunk 13 above)
Comment (rudyflores) [reply packages/cli/README.md:9]: In this case we point to external docs for more information on both methods, I meant to say "Use these methods when ..." as a disclamer before showing the link to external docs, so I think a separation is not necessary
